input.css

```diff
@@ -140,14 +140,6 @@ pre {
   text-align: right;
 }
 
-/* Get background for Ensign Developer Experience section to span the full width */
```

Comment:
This CSS class is no longer used since changes have been made to the Ensign page.